core/drivers/crypto/se050/glue/i2c_stm32.c

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) Foundries Ltd. 2022 - All Rights Reserved
+ * Author: Jorge Ramirez-Ortiz <jorge@foundries.io>
+ */
+
+#include <drivers/stm32_i2c.h>
+#include <i2c_native.h>
+#include <kernel/boot.h>
+#include <kernel/dt.h>
+#include <libfdt.h>
+#include <phNxpEsePal_i2c.h>
```

Comment:
Where is this file supposed to be?

Reply:
it is in the Plug-and-Trust repo (it is an external dependency)
https://github.com/foundriesio/plug-and-trust